app/models/spree/order_decorator.rb

```diff
@@ -24,24 +24,6 @@
   before_save :update_shipping_fees!, if: :complete?
   before_save :update_payment_fees!, if: :complete?
 
-  checkout_flow do
-    go_to_state :address
-    go_to_state :delivery
-    go_to_state :payment, :if => lambda { |order|
-      # Fix for #2191
-      if order.shipping_method.andand.require_ship_address and
```

Comment:
the reason behind all this is that in Spree 2.0 we have `order.shipments` which have a `shipping_method`.